organizations/schronisku-dla-zwierzat-w-olsztynie.yaml

```
@@ -0,0 +1,27 @@
+---
+nazwa: "Schronisko dla Zwierząt w Olsztynie"
+
+adres: schronisku-dla-zwierzat-w-olsztynie
+
+strona: https://schronisko.olsztyn.eu
+
+krs: 0000226639
```

```suggestion
krs: "0000355809"
```
Podany KRS to był Medor